src/components/PageTitle.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import styles from '../styles/breadcrumb'
+import { translate } from '../lib/I18n'
+
+const PageTitle = ({pathname, t}) => {
+  let title
+  switch (pathname) {
+    case String(pathname.match(/^\/settings/)) :
+      title = 'mobile.settings'
```

Comment:
We started to categorize locales since files-v3, it's much easier to read and to use. I also prefer the categorized way imho.